server/settings/create.py

```diff
@@ -9,6 +9,22 @@ class CreatorModel(BaseSettingsModel):
         default_factory=list,
     )
 
+def review_node_types_enum():
+    return [
+        {"label": "OpenGL", "value": "opengl"},
+        {"label": "Flipbook", "value": "flipbook"}
+    ]
+
+class CreateReviewModel(BaseSettingsModel):
+    enabled: bool = SettingsField(title="Enabled")
+    default_variants: list[str] = SettingsField(
+        title="Default Products",
+        default_factory=list,
+    )
+    node_type: str = SettingsField(
+        Title="Default Node Type",
```

Comment:
No idea why the title doesn't show the word `Default`
![image](https://github.com/user-attachments/assets/30766060-aafd-488d-a022-007bcf0d0116)

Reply:
It's the capital `T` ;)
```suggestion
        title="Default Node Type",
```